### PR TITLE
Update to support Rake 11

### PR DESCRIPTION
**Pull request by tbrisker**
Before submitting a PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it)
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Added an entry to the [Changelog](CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../CONTRIBUTING.md#changelog-entry-format).
* [ ] All tests are passing.
* [x] The new code doesn't generate RuboCop offenses.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.

[1]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

See https://github.com/ruby/rake/blob/master/History.rdoc#1100--2016-03-09

### CODE REVIEW

**Comment by bbatsov** [comment]
Will this work on older rakes?

**Comment by hsbt** [comment]
@bbatsov Yes. `last_comment` is deprecated for a long time. `last_decription` is also works Rake 10.x series. see. https://github.com/ruby/rake/blob/ed197e56cc626725f02431d909e02e048244488e/lib/rake/task_manager.rb#L8

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 ### Bug fixes
 
+* Replace Rake::TaskManager#last_comment with Rake::TaskManager#last_description for Rake 1.11 compatibility. ([@tbrisker][])
```
Comment (hsbt) [inline CHANGELOG.md]: s/Rake 1.11/Rake 11/ ?

Comment (tbrisker) [inline CHANGELOG.md]: Oops, my mistake. will fix.

**Comment by alexdowad** [comment]
Why not bump Rake to the latest version in the same commit?

**Comment by tbrisker** [comment]
@alexdowad will do. Pushing new version with rake ~> 11.0.

**Comment by stephenrjohnson** [comment]
:+1: 

Hopefully this is released soon.

**Comment by bkarstaedt** [comment]
+1

updating/deploying many apps because of a bundler-audit warning is currently failing because of this bug :/

**Comment by bbatsov** [comment]
I'll try to cut a new release later today.

**Pull request merged**